pymc3/tests/test_variational_inference.py

```diff
@@ -835,6 +835,20 @@ def test_sample_replacements(binomial_model_inference):
     assert sampled.shape[0] == 101
 
 
+def test_var_replacement():
+    X_mean = np.linspace(0, 10, 10, dtype='float32')
```

Comment:
This is still causing test fail. Why not use `dtype=theano.config.floatX`?

Reply:
Probably